app/src/main/res/drawable/baseline_star_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="12dp" android:tint="#6D6D80"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="12dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="@android:color/white" android:pathData="M12,17.27L18.18,21l-1.64,-7.03L22,9.24l-7.19,-0.61L12,2 9.19,8.63 2,9.24l5.46,4.73L5.82,21z"/>
```

Comment:
never use @android:color
create your own white and use it

Reply:
Done